File: Data Structures & Algorithms/duplicate-integer/submission-0.cpp
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        std::unordered_map<int,int> dict;
        for (int i : nums){
            dict[i]++;
            if (dict[i] > 1){
                return true;
            }
        }   
        return false;
    }
};

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        max_length = 0
        set_nums = set(nums)
        for i in set_nums:
            if i-1 in set_nums:
                continue
            current = i
            length = 1
            while current+1 in set_nums:
                current+=1
                length+=1
            if length > max_length:
                max_length = length
        return max_length